### ISSUE TITLE
'caf' command returns no further information...

### ISSUE THREAD

**Issue by panther2** [openscope/openscope#358]
... than `cleared to destination as filed`
So for departing planes it is difficult to determine their departure.

I open this issue as #320 addresses a KORD-specific subject and this new one appears on all airports I have tested so far with version 5.0.0.

So to get an idea where departing airplanes will move after taking off, I need to command `say route` before or after `taxi`.

At KSFO for example planes according to the flight strip are going to QUIET7, MOLEN7, OFFSH9 and PORTE7 ... but which of them?
Only after `say route` I learn that QUIET7.rbl, MOLEN7.eni, OFFSH9.sxc and PORTE7.czq appear to be the only exits, that are in use (what about the others?). During the flight the flight strip only indicates for example QUIET7 (bottom) and the next waypoint (top).

I am sorry... maybe this all is intentional, but there have been so many changes during the last weeks that I feel a bit lost on how departures are handled now. Do I really have to include `say route` to every departing procedure, now?

**Comment by n8rzz** [comment]
These are the various states of the flight strip and the corresponding command to get them to that state:
<img width="757" alt="screen shot 2017-04-22 at 1 21 32 pm" src="https://cloud.githubusercontent.com/assets/3325985/25306998/b6f1ed1e-275e-11e7-9095-3babed3295bf.png">

You should be able to see the exit of a SID after an aircraft has been told to `caf`.  If that's not the case, then we have a bug.

Please keep brining them up as you see them.  There were _a lot_ of changes in this release and there are likely to be ore bugs that we have not discovered yet.  And thats ok.  We just need to identify them and the steps to reproduce them and then we can begin fixing them.

**Comment by panther2** [comment]
> You should be able to see the exit of a SID after an aircraft has been told to caf. If that's not the case, then we have a bug.

Indeed this is no longer the case. 
Also on KORD only `ord1.noony` is used (identified by `sr` or just watching the planes fly), and
on KSFO only the above mentioned exits are used (according to `sr` and/or watching). I will post screenshots/casts soon.

**Comment by n8rzz** [comment]
You could also look at the airport json files, specifically the soawnPatterns section.  Look for the departures. It is possible there is only one defined at the moment. Once I get to a computer later tonight I'll check it out.

**Comment by Fechulo** [comment]
@n8rzz That's correct, since there's only one departure procedure only one spawnPattern was defined when they were added. before v5.0.0 this didn't really matter because the code ignored the exitPoint specified in the spawnPatterns, but now that it works as expected, all aircraft want to fly the same transition.  KORD needs a lot of changes, which I'll get to eventually.

**Comment by panther2** [comment]
Thanks, I will add the KSFO cast nevertheless:
![oscokl2](https://cloud.githubusercontent.com/assets/20812775/25307344/e7458980-279f-11e7-9fd0-2c8b1ae714ef.gif)

You can see that `caf` does not have any informatory effect, it is only after Takeoff that the next waypoint is indicated, still not giving any information about the used exit. I am fine if this is by design, but it is irritating nevertheless.

**Comment by n8rzz** [comment]
@erikquinn what are your thoughts on how this should display and how the readbacks should be structured?

**Comment by erikquinn** [comment]
The readback I think is just too distracting when it's all the way done out and proper... Normally I'd be for maximum realism, but it makes me cringe more to see clearance delivery verbage being done while I'm working approach control... so the readback not containing the procedure I think is fine. People have probably come to use it for that purpose, but the strips should be showing the entire route (again, goes back to #18/#295). For the moment, the resolution should be to fix the current strips so they show the procedure's entire route string, minus the airport section (so `OFFSH9.SXC` and `QUINN.BDEGA2`) from the very beginning. And if we forget which procedure an aircraft is on, we have to find the strip (via selecting them and finding the highlighted strip) or ask the pilot to say their route.

**Comment by erikquinn** [comment]
Created #359 to solve the strip issue and #360 to get more than one departure spawn pattern into o'hare. 😄 

Closing as duplicate of those issues.
